### PR TITLE
Updated managedservices for version 2019-06-01

### PR DESCRIPTION
**Pull request by rajivnandivada**
Updated release notes, tests.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/managedservices/azure-mgmt-managedservices/tests/test_mgmt_managedservices.py [hunk 1]
```diff
@@ -19,50 +19,60 @@ def setUp(self):
...
         properties = json.loads((regdef_string))
+        sleep=15
 
         #create definition
-        defresponse = self.client.registration_definitions.create_or_update(
-                    definitionid, api_version, scope, properties)
-        definitionresourceid = defresponse.id
-        self.assertIsNotNone(defresponse)
-        self.assertEqual(definitionid, defresponse.name)
+        response = self.client.registration_definitions.create_or_update(definitionid, api_version, scope, properties)
```
Comment (lmazuel) [inline sdk/managedservices/azure-mgmt-managedservices/tests/test_mgmt_managedservices.py]: That's not the correct way to use this API:
```python
poller = self.client.registration_definitions.create_or_update(definitionid, api_version, scope, properties)
poller.wait()
```

Please apply this pattern to all LRO operations that doesn't have return type

Comment (lmazuel) [inline sdk/managedservices/azure-mgmt-managedservices/tests/test_mgmt_managedservices.py]: api_version should not be a parameter of the operation, but a constant in the main client. This is likely a Swagger issue that wasn't caught by the Swagger reviewer.

Comment (lmazuel) [reply sdk/managedservices/azure-mgmt-managedservices/tests/test_mgmt_managedservices.py]: merge from https://github.com/Azure/azure-sdk-for-python/tree/restapi_auto_managedservices/resource-manager and you should not have to define api_version ever again

**Comment by rajivnandivada** [comment]
Closing this one for https://github.com/Azure/azure-sdk-for-python/pull/6235

**Pull request closed without merging**